pkg/controllers/nodeclaim/expiration/controller.go

```diff
@@ -63,15 +63,10 @@ func (c *Controller) Reconcile(ctx context.Context, nodeClaim *v1.NodeClaim) (re
 	}
 	// 4. The deletion timestamp has successfully been set for the NodeClaim, update relevant metrics.
 	log.FromContext(ctx).V(1).Info("deleting expired nodeclaim")
-	metrics.NodeClaimsDisruptedCounter.With(prometheus.Labels{
-		metrics.TypeLabel:     metrics.ExpirationReason,
+	metrics.NodeClaimsDisruptedTotal.With(prometheus.Labels{
+		metrics.ReasonLabel:   metrics.ExpirationReason,
```

Comment:
Should this be changed from `Expiration` to `Expired`?

Reply:
Let me rename the constant. It's value is `expired` under the hood.